### PR TITLE
Completed typescript-conditionals exercise - Shea Pone

### PR DESCRIPTION
**Pull request by SheaPone**
Screenshot 2024-07-16 at 9 28 03 PM (2)

### CODE REVIEW

File: typescript-conditionals/main.ts [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  if (person.age >= 16) {
+    return true;
+  } else {
+    return false;
+  }
+}
+console.log('isOldEnoughToDrive:', isOldEnoughToDrive(shea));
+
+function isOldEnoughToDrinkAndDrive(person: Person): any {
+  if (person.age >= 0) {
```
Comment (hesam) [inline typescript-conditionals/main.ts]: they want you to use AND operator for the two conditions (drinking and driving).

**Review by hesam — CHANGES_REQUESTED** [review]
See comment in code

**Comment by SheaPone** [comment]
Screenshot 2024-07-17 at 7 22 15 PM

**Review by hesam — APPROVED** [review]
Great!

**Pull request merged**